system-contract-dapp-playground/src/api/hedera/hts-interactions/tokenCreateCustom-interactions/index.ts

```diff
@@ -73,7 +75,8 @@ export const createHederaFungibleToken = async (
   treasury: string,
   inputKeys: ICommonKeyObject[],
   msgValue: string,
-  feeTokenAddress?: string
+  feeTokenAddress?: string,
+  feeTokenAmount?: number
```

Comment:
small nit, should we use `feeAmount` to be consistent with the rest of the PR?

Reply:
Gotcha I don't know why I left it as feeTokenAmount! Good catch thanks!